### ISSUE TITLE
[ACLs] Show error page when access to project is denied by backend

### ISSUE THREAD

**Issue by gino-m** [google/ground-platform#609]
When a `PermissionDenied` error is returned by server, client should redirect to an error page. The sign-in page can be refactored and used for this purpose.

**Comment by gino-m** [comment]
@parulraheja98 We spoke, queued after #743.

**Comment by gino-m** [comment]
Duplicate of #931.
